why3/README.md

```diff
@@ -1,7 +1,7 @@
 # Why3 
 
 A Rust library furnishing the Why3 AST used by the Creusot verifier, and its associated pretty printer. 
-The library supports three input formats to why3: WhyML (`.mlw`), coma (`.coma`) and Coma (`.coma`).
+The library supports three input formats to why3: WhyML (`.mlw`), Coma (`.coma`).
```

Comment:
"three" -> "two"?

Reply:
Good catch!

Reply:
technically it _is_ 3 since its `mlcfg`, `coma` and `mlw`.